### ISSUE TITLE
HTTP error with local builds

### ISSUE THREAD

**Issue by aerwin** [maqetta/maqetta#2525]
@JonFerraiolo saw this over the weekend. And, now when I'm trying to do a local build against the latest code on the server repository, I'm seeing it as well. Basically, the build completes successfully, but when going to http://localhost:50000/maqetta, I see:

```
HTTP ERROR 404

Problem accessing /maqetta. Reason:

    ProxyServlet: /maqetta
Powered by Jetty://
```

I know launching using the Eclipse launch scripts got broken last week. And, there were some dependency issues that @peller and @childsb had to work out to fix that. Does something need to be done to the main launch scripts in the build output as well?

Sending to @childsb as a start since he put in the fix for the Eclipse launch scripts.

(I'm on Mac and no server errors in console.)

**Comment by billreed63** [comment]
I was seeing something this last week with Adams addtion of 
org.apache.commons.httpclient
maqetta.core.server: META-INF/MANIFEST.MF

Maybe the package is missing.

**Comment by peller** [comment]
I think @childsb said he got this working?

**Comment by childsb** [comment]
Fixed here: 241a4bb912b66c72c830bed76738bff7b74ce236
